tests/integration/golang/admin/namespace/create_test.go

```diff
@@ -0,0 +1,147 @@
+//go:build integration
+
+package namespace
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/common"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+	"github.com/G-Research/fasttrackml/pkg/ui/admin/request"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type CreateNamespaceTestSuite struct {
+	suite.Suite
+	helpers.BaseTestSuite
+}
+
+func TestCreateNamespaceTestSuite(t *testing.T) {
+	suite.Run(t, new(CreateNamespaceTestSuite))
+}
+
+func (s *CreateNamespaceTestSuite) SetupTest() {
+	s.BaseTestSuite.SetupTest(s.T())
+}
+
+func (s *CreateNamespaceTestSuite) Test_Ok() {
+	defer func() {
+		assert.Nil(s.T(), s.NamespaceFixtures.UnloadFixtures())
+	}()
+	_, err := s.NamespaceFixtures.CreateNamespace(context.Background(), &models.Namespace{
+		ID:                  1,
+		Code:                "default",
+		DefaultExperimentID: common.GetPointer(int32(0)),
+	})
+	assert.Nil(s.T(), err)
+
+	requests := []request.Namespace{
+		{
+			Code:        "test2",
+			Description: "test namespace 2 description",
+		},
+		{
+			Code:        "test3",
+			Description: "test namespace 3 description",
+		},
+	}
+	for _, request := range requests {
+		assert.Nil(
+			s.T(),
+			s.AdminClient.WithMethod(
+				http.MethodPost,
+			).WithRequest(
+				request,
+			).DoRequest("/namespaces"),
+		)
+	}
+
+	namespaces, err := s.NamespaceFixtures.GetNamespaces(context.Background())
+	assert.Nil(s.T(), err)
+
+	// Check that the namespace has been created
+	for _, namespace := range namespaces {
```

Comment:
I think this could be done more simply with `Contains`, eg: `assert.Contains(s.T(), namespaces, model.Namespace{Code: testNamespace.Code, Description: testNamespace.Description})`

Reply:
They are going to have different fields content except for code and description, like ID, createdAt, etc… So they won't be found.

Reply:
we can hide that in `helpers` and have more readable  code.

Reply:
I'll do that